kebechet/exception.py

```diff
@@ -62,3 +62,27 @@ class WebhookPayloadError(KebechetException):
 
     Generally the payload is empty or not of proper json format.
     """
+
+
+class CannotFetchPRError(KebechetException):
+    """
+    An exception raised if there OGR cannot fetch the PR's for a repo.
```

Comment:
Docstring starts with a single sentence, coala/linters will argue here:

```python
    """An exception raised if there OGR cannot fetch the PR's for a repo.

    Generally the token is invalid or doesn't have access to the repo.
    """
```

Here and bellow